packages/sanity-studio/src/utils/defineIcon.ts

```diff
@@ -0,0 +1,32 @@
+import { ComponentType } from "react";
```

Comment:
Can this be a normal schema export under an `/src/schemas` folder instead of a utility? 

This way I can do this from user-land once I import the icon schema add it to my studio schemas:

```
{
   name: "myIcon",
   type: "icon"
}

```